plugins/utils/todo-comments.nix

```diff
@@ -76,8 +76,8 @@ in
               TODO = { icon = " "; color = "info"; };
               HACK = { icon = " "; color = "warning"; };
               WARN = { icon = " "; color = "warning"; alt = [ "WARNING" "XXX" ]; };
-              PERF = { icon = " "; alt = [ "OPTIM" "PERFORMANCE" "OPTIMIZE" ]; };
```

Comment:
This looks like the defaults section for the plugin, the defaults I see in their config uses a different icon: 
`PERF = { icon = " ", alt = { "OPTIM", "PERFORMANCE", "OPTIMIZE" } },`